File: 30_days_of_code/3-conditional-statements/cpp/conditional-statements.cpp
#include <iostream>
using namespace std;

int main(){
  int N;
  cin >> N;

  // If  is odd, print Weird
  if((N & 1) == 1)
    cout << "Weird" << endl;
  //   If  is even and in the inclusive range of 2 to 5, print Not Weird
  else if(N >= 2 && N <= 5)
    cout << "Not Weird" << endl;
  //   If  is even and in the inclusive range of  6 to 20, print Weird
  else if(N >= 6 && N <= 20)
    cout << "Weird" << endl;
  //   If  is even and greater than 20, print Not Weird
  else if(N > 20)
    cout << "Not Weird" << endl;


  return 0;
}

File: 30_days_of_code/3-conditional-statements/py/conditional-statements.py
#!/bin/python

import sys

n = int(raw_input().strip())

# If  is odd, print Weird
if (n & 1) == 1:
    print "Weird"
#   If  is even and in the inclusive range of 2 to 5, print Not Weird
elif n in range(2, 6):
    print "Not Weird"
# If  is even and in the inclusive range of  6 to 20, print Weird
elif n in range(6, 21):
    print "Weird"
#  If  is even and greater than 20, print Not Weird
elif n > 20:
    print "Not Weird"
